ironfish/src/rpc/routes/wallet/multisig/createTrustedDealerKeyPackage.ts

```diff
@@ -76,42 +78,46 @@ routes.register<
       outgoingViewKey,
       proofAuthorizingKey,
       keyPackages,
-    } = multisig.generateAndSplitKey(minSigners, identities)
+    } = multisig.generateAndSplitKey(request.data.minSigners, identities)
 
-    const latestHeader = node.chain.latest
     const createdAt = {
-      hash: latestHeader.hash,
-      sequence: latestHeader.sequence,
+      hash: node.chain.latest.hash,
+      sequence: node.chain.latest.sequence,
     }
 
-    const encoder = new Base64JsonEncoder()
-    const participantAccounts = keyPackages.map(({ identity, keyPackage }) => ({
-      identity,
-      account: encoder.encode(
-        {
-          name: identity,
-          version: ACCOUNT_SCHEMA_VERSION,
-          createdAt,
-          spendingKey: null,
-          viewKey,
-          incomingViewKey,
-          outgoingViewKey,
-          publicAddress,
-          proofAuthorizingKey,
-          multisigKeys: {
-            identity,
-            keyPackage,
-            publicKeyPackage,
-          },
-        },
-        {
-          encryptWith: {
-            kind: 'MultisigIdentity',
-            identity: Buffer.from(identity, 'hex'),
-          },
+    const participants = keyPackages.map(({ identity, keyPackage }) => {
+      const account: AccountImport = {
+        name: identity,
+        version: ACCOUNT_SCHEMA_VERSION,
+        createdAt,
+        spendingKey: null,
+        viewKey,
+        incomingViewKey,
+        outgoingViewKey,
+        publicAddress,
+        proofAuthorizingKey,
+        multisigKeys: {
+          identity,
+          keyPackage,
+          publicKeyPackage,
         },
-      ),
-    }))
+      }
+
+      const encoder = new JsonEncoder()
```

Comment:
This was `Base64JsonEncoder` originally, double-checking that you intended to change it to `JsonEncoder`

Reply:
yeah, it's a little tricky but changing this to JsonEncoder prevents it from being double base64 encoded.

The encryption base64 encodes and adds the prefix to the encrypted result. It means here we use the JsonEncoder to just avoid double base64 which is unnecessary.